sample-apps/donet-sample-app/Controllers/MetricEmitter.cs

```diff
@@ -76,33 +82,52 @@ public MetricEmitter()
                 "By",
                 "Keeps a sum of the total amount of bytes sent while the application is alive");
 
-            apiLatencyRecorder = meter.CreateHistogram<double>(latencyMetricName,
-                 "ms", 
-                 "Measures latency time in buckets of 100 300 and 500");
-
-            totalTimeSentObserver = meter.CreateCounter<long>(totaltimealiveMetricName,
-                "ms",
-                "Measures the total time the application has been alive");
+            meter.CreateObservableGauge(cpuUsageMetricName, () => cpuUsage, 
+                "1",
+                "Cpu usage percent");
 
-            meter.CreateObservableUpDownCounter(totalHeapSizeMetricName, () => totalHeapSize, 
+            meter.CreateObservableUpDownCounter(totalHeapSizeMetricName, () => { 
+                    return new List<Measurement<long>>()
+                    {
+                        new Measurement<long>(UpDowntick++ * 10),
+                    };
+                }, 
                 "By",
                 "The current total heap size”");  
 
-            meter.CreateObservableCounter(totalThreadsMetricName, () => totalThreadSize,
-                "1",
-                "The current total number of threads”");
+            apiLatencyRecorder = meter.CreateHistogram<double>(latencyMetricName, 
+                 "ms", 
+                 "Measures latency time in buckets of 100 300 and 500");
 
-            meter.CreateObservableGauge(cpuUsageMetricName, () => cpuUsage,
+            totalThreadsObserver = meter.CreateUpDownCounter<int>(totalThreadsMetricName, 
                 "1",
-                "Cpu usage percent");
+                "The total number of threads active”");
+
+            totalTimeSentObserver = meter.CreateCounter<int>(totaltimealiveMetricName,
+                "ms",
+                "Measures the total time the application has been alive");
+            
 
+            totalTimeSentObserver.Add(totaltime,
+                new KeyValuePair<string, object>("signal", "metric"),
+                new KeyValuePair<string, object>("language", "dotnet"),
+                new KeyValuePair<string, object>("metricType", "request"));
```

Comment:
This set of key pairs looks to be reused quite a lot. Can we put store them in a variable for resuse?